docs/getting-started/app-custom-theme.md

```diff
@@ -234,3 +234,73 @@ const LogoFull = () => {
   return <img src={MyCustomLogoFull} />;
 };
 ```
+
+## Custom Homepage
+
+In addition to a custom theme, a custom logo, you can also customize the
+homepage of your app. To do that we need to go through a few steps.
+
+### Setting up the Home Page
+
+1. Create a Home Page Component that will be used for composition.
+
+`packages/app/src/components/home/HomePage.tsx`
+
+```tsx
+import React from 'react';
+
+export const HomePage = () => {
```

Comment:
I know it's already merged, but shouldn't this be an element instead of component? Similar to how it works for EntityPage
https://backstage.io/docs/plugins/composability#porting-entity-pages